src/components/MsgPanels.js

```diff
@@ -33,33 +35,48 @@ export default class MsgPanel extends React.Component {
             loading,
             error,
             success,
+            btn,
```

Comment:
Unused params...

Reply:
Sorry, `btn` is used, `testid`, `transactions` are not.
But is `btn` nessessary? We have `dismissable`, `dismissed`, `onDismiss`...

Reply:
i usethe  `btn` so we can have an `X` close BTN only, instead of an `OK` BTN & an `X` as well. 
also, this way i did not have to rewrite and check all the things that use the MsgPanels